Applications/TransferLearning/Draw_Classification/jni/main.cpp

```diff
@@ -363,17 +371,16 @@ int testModel(const char *data_path, const char *model) {
 
 fail_info_release:
   ml_tensors_info_destroy(in_info);
-
-  status = ml_pipeline_stop(pipe);
+  ml_pipeline_stop(pipe);
```

Comment:
For future you can consider the below style to avoid coverity issues in cases like these.
```c++
new_status = ml_pipeline_stop(pipe);
if (new_status != ML_ERROR_NONE && status == ML_ERROR_NONE)
  status = new_status;
```

Let's see if the current code makes an issue (of ignoring the return value).

Reply:
I think coverity will make a return value issues.
I applied your suggestion

Reply:
This style returns the first error that occurred in the code (as other errors can be caused by the first error).
If Coverity gives the issues of `return value ignored`, then it can be marked false positive.